cleanlab/count.py

```diff
@@ -423,6 +380,8 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
       If `return_indices_of_off_diagonals` is ``True``, `confident_joint_counts` is the first element of returned tuple
       and second element is another array of indices counted in off-diagonals of confident joint.
 
+    Note: if `return_indices_of_off_diagonals` is set as True, this function instead returns a tuple `(confident_joint, indices_off_diagonal)`
+    where `indices_off_diagonal` is an array and each array contains the indices of examples counted in off-diagonals of confident joint.
     Note
```

Comment:
Don't you need a space between these?  Have you built the docs based on your PR and verified they look good?

Reply:
```suggestion
    where `indices_off_diagonal` is an array and each array contains the indices of examples counted in off-diagonals of confident joint.
    
    Note
```

Reply:
I'll double check